app/src/main/java/no/usn/mob3000/ui/components/base/TopNavBar.kt

```diff
@@ -50,9 +50,10 @@ fun TopNavbar(
     topBarActions: @Composable (RowScope.() -> Unit) = {},
 ) {
     TopAppBar(
-        title = { if (showTitle) Text(stringResource(currentScreen.title)) },
+        title = { if (showTitle) Text(stringResource(currentScreen.title),color = MaterialTheme.colorScheme.onSurface) },
         colors = TopAppBarDefaults.mediumTopAppBarColors(
-            containerColor = NavbarBackground
+            containerColor = MaterialTheme.colorScheme.surface,
+            titleContentColor = MaterialTheme.colorScheme.onSurface
```

Comment:
This did not have a title content color defined before. Are you certain this is necessary? See [this comment](https://github.com/frigvid/mob3000-group11/pull/27/files#r1847123849).